index.html

```diff
@@ -0,0 +1,133 @@
+                <li><a href="#" class="navbar-cc-link">CC Campaign</a></li>
+            </ul>    
+        </div>
+    </nav>
+    <!--THE MAIN PAGE SECTION-->
+    <section class="first-section">
+        <div class="page-head">
+            <i class="fas fa-bars fa-2x hambuger black"></i>
+            <div><p class="orange hellos">"Hello! Sharing world"</p></div>
+            <h1 class="orange">CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
+            <p class="orange hello">"Hello! Sharing world"</p>
+        </div>
+        <p class="black intro">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea</p>
+        <h2 class="black date">2015.10.15(THUR) ~ 16(FRI)</h2>
+        <p class="black location">@National Museum of Korea, Art Center Nabi and more</p>
+    </section>
+    <!--THE MAIN PROGRAM SECTION-->
+    <section class="second-section">
+        <h2 class="white">Main Program</h2>
+        <div class="second-section-container">
+            <div class="prog-feature">
+                <img src="./assets/icon-1.png" alt="Lecturer">
+                <h4 class="orange">Lecture</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+            <div class="prog-feature">
+                <img src="./assets/icon-2.png" alt="CC Exhibition">
+                <h4 class="orange">Exhibition</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+            <div class="prog-feature">
+                <img src="./assets/icon-3.png" alt="Forum">
+                <h4 class="orange">Forum</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+            <div class="prog-feature">
+                <img src="./assets/icon-4.png" alt="Workshop">
+                <h4 class="orange">Workshop</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+            <div class="prog-feature">
+                <img src="./assets/icon-5.png" alt="CC Ignite">
+                <h4 class="orange">CC Ignite</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+        </div>
+        <a href="#" class="white logo-link-1">Join CC Global Summit 2015</a>
+        <a href="#" class=" logo-link-2 white">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <!--THE FEATURED SPEAKERS SECTION-->
+    <section class="third-section">
+        <h2>Featured Speakers</h2>
+        <aside id="my-popup"></aside>
+    </section>
+    <footer>
+        <div class="partners">
```

Comment:
- [ ] Good job using a footer for mobile size. Now let's try to use the same footer for both pages (the footer should be exactly the same). 🦃 
- [ ] Look at this bug on about page footer, the text looks a little overhead. Please work on it. 🌵 
![image](https://user-images.githubusercontent.com/20563252/128196793-9fa589af-dace-42d1-9f39-1e9e6ebcbc64.png)

- [OPTIONAL] It would be excellent if the background color for the desktop size footer would be a dark color. For mobile is ok on white.